static/src/stylesheets/module/email/_form.scss

```diff
@@ -0,0 +1,220 @@
+// TODO These mixins are here while we run the A/B test as the styles are custom to the email form but if they become perm then
+// they should be made reusable
+
+@mixin rounded-input--inline-button() {
+    &text-input {
+        border-right: 0;
+        @include border-radius(1000px 0 0 1000px);
+        float: left;
+        width: 65%;
+    }
+
+    &submit-input {
+        font-size: 12px;
+        @include border-radius(0 1000px 1000px 0);
+        border-left: 0;
+        float: left;
+        margin-right: 0;
+        padding-left: 10px;
+        width: 35%;
+    }
+}
+
+.email-sub {
+    @include clearfix();
+}
+
+.email-sub__ { // Email subscription component namespace
+
+    /**
+     * Base Styles
+     */
+
+    &iframe {
```

Comment:
Yeah I can see the benefit of using it. I'm not sure to be honest, just
wish we had scoped CSS :unamused:

On Fri, 13 Nov 2015 at 11:05 Gareth Trufitt <notifications@github.com>
wrote:

> In static/src/stylesheets/module/email/_form.scss
> <https://github.com/guardian/frontend/pull/11114#discussion_r44771629>:
>
> > +        padding-left: 10px;
> > +        width: 35%;
> > +    }
> > +}
> > +
> > +.email-sub {
> > +    @include clearfix();
> > +}
> > +
> > +.email-sub__ { // Email subscription component namespace
> > +
> > +    /**
> > +     * Base Styles
> > +     */
> > +
> > +    &iframe {
>
> Seemed cleaner to extend out of a namespace, but that's a good point. I'll
> pull this nesting out.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/11114/files#r44771629>.
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396